source/features/user-local-time.tsx

```diff
@@ -101,7 +101,13 @@ function init(): void {
 
 		// Adding the time element might change the height of the hovercard and thus break its positioning
 		const hovercardHeight = hovercard.offsetHeight;
-		select('div.d-flex.mt-3 > div.overflow-hidden.ml-3:not([data-hovercard-type="contributors"]', hovercard)!.append(container);
+
+		const userHovercard = select('div.d-flex.mt-3 > div.overflow-hidden.ml-3', hovercard)!;
+		if (!userHovercard) {
```

Comment:
When does this happen?

Reply:
#2641 
#2549

Reply:
Didn’t #2641 fix that? We try to avoid these kinds of checks because they make actual errors silent (i.e. when GitHub changes the classes)

Reply:
Nope

Reply:
This check is too late. Line 86 should be extended to include some check for: "is this hovercard for single user?"

Reply:
Explain. I was having a hard time with vscode until last night. I will try to see if there is a better later

Reply:
I mean this change should be undone, there’s a lot of code that is run unnecessarily before this. 

It’s better to check whether we need to run this function at all. Line 86 already has checks for this
- is empty? Return
- is timezone element already there? Return

We should add:

- is not a user hover card? Return

Reply:
Ok lets leave it for now